index.html

```diff
@@ -47,11 +49,172 @@ <h1 class="h-title">I’m ROBERTO. Glad to see you!</h1>
+          <ul class="canopy">
+            <li class="cano"><a href="#" class="can">CANOPY</a></li>
+            <li><a href="#">Back End Dev</a></li>
+            <li><a href="#">2015</a></li>
+          </ul>
+  
+      <p class="text">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+  
+      <ul class="ul">
+          <li class="tag">
+              <a href="">HTML</a>
+          </li>
+          <li class="tag">
+              <a href="" >CSS</a>
+          </li>
+          <li class="tag">
+              <a href="">JavaScript</a>
+          </li>
+      </ul>
+  
+      <a href="#" class="button project">See Project</a>
+       </div>
+  
       
-     
-      <section>
+  
+      </div>
+      <div class="card works">
+  
+    <img src="img/Snapshoot2.png" alt="a_screentshot" class="pic">
+  
+    <div class="text-card">
+        <h2 class="titleh2">Tonic</h2>
+        <ul class="canopy">
+          <li class="cano"><a href="#" class="can">CANOPY</a></li>
+          <li><a href="#">Back End Dev</a></li>
+          <li><a href="#">2015</a></li>
+        </ul>
+  
+    <p class="text">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+  
+    <ul class="ul">
+        <li class="tag">
+            <a href="">HTML</a>
+        </li>
+        <li class="tag">
+            <a href="" >CSS</a>
+        </li>
+        <li class="tag">
+            <a href="">JavaScript</a>
+        </li>
+    </ul>
+  
+    <a href="#" class="button project">See Project</a>
+     </div>
+  
+    
+  
+    </div>
+    <div class="card works">
+  
+    <img src="img/Snapshoot3.png" alt="a_screentshot" class="pic">
+  
+    <div class="text-card">
+      <h2 class="titleh2"> Multi-Post Stories</h2>
+        <ul class="canopy">
+          <li class="cano"><a href="#" class="can">CANOPY</a></li>
+          <li><a href="#">Back End Dev</a></li>
+          <li><a href="#">2015</a></li>
+        </ul>
+  
+    <p class="text">
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+  
+    <ul class="ul">
+        <li class="tag">
+            <a href="">HTML</a>
+        </li>
+        <li class="tag">
+            <a href="" >CSS</a>
+        </li>
+        <li class="tag">
+            <a href="">JavaScript</a>
+        </li>
+    </ul>
+  
+    <a href="#" class="button project">See Project</a>
+     </div>
+  
+    
+  
+    </div>
+    </section>
+  <div class="myself-section">
+    <div class="title">
+        <h3 class="h-title">About Myself</h3>
+      </div>
+      <div class="parrafo">
+        <p>Hello I’m a software developer! I can help you build a product , feature or website.
+           Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+             don’t hesitate to contact me.
+        </p>
+      </div>
```

Comment:
- [ ] [OPTIONAL] Good work developing your About myself section ✔️. However, please kindly consider wrapping your  About myself section using a semantic `<section>` tag instead of using a `<div>` tag 🤓.